src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -86,6 +84,9 @@ def initialize_module_for_quantization(
 
     offloaded = False
     if is_module_offloaded(module):
+        from accelerate.hooks import add_hook_to_module, remove_hook_from_module
+        from accelerate.utils import PrefixedDataset
```

Comment:
Do we need to have an error message to `pip install accelerate` if not installed?